samples/boards/nrf52/mesh/onoff-app/prj.conf

```diff
@@ -0,0 +1,96 @@
+#CONFIG_INIT_STACKS=y      # if set, lots of noise from mesh/adv.c: STACK_ANALYZE
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_BOOT_BANNER=y
+CONFIG_BUILD_TIMESTAMP=y
+
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_CTLR_DUP_FILTER_LEN=0
+CONFIG_BT_CTLR_LE_ENC=n
+CONFIG_BT_CTLR_LE_PING=n
+CONFIG_BT_CTLR_DATA_LENGTH=n
+CONFIG_BT_CTLR_PHY=n
+CONFIG_BT_CTLR_CHAN_SEL_2=n
+CONFIG_BT_CTLR_MIN_USED_CHAN=n
+CONFIG_BT_CTLR_ADV_EXT=n
+CONFIG_BT_CTLR_PRIVACY=n
+
+CONFIG_BT_PERIPHERAL=y
+
+CONFIG_BT=y
+CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_BT_RX_BUF_COUNT=30
+CONFIG_BT_L2CAP_RX_MTU=69
+CONFIG_BT_L2CAP_TX_MTU=69
+CONFIG_BT_L2CAP_TX_BUF_COUNT=8
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_LOW_POWER=n
+CONFIG_BT_MESH_FRIEND=n
+CONFIG_BT_MESH_FRIEND_QUEUE_SIZE=16
+CONFIG_BT_MESH_ADV_BUF_COUNT=20
+
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_GATT_PROXY=y
+CONFIG_BT_MESH_CFG_CLI=y
+CONFIG_BT_MESH_HEALTH_CLI=y
```

Comment:
Are you actually using or planning to use the configuration & health client models in the application? Those are usually reserved for provisioner-style devices or test apps (like the shell).

Reply:
Ok,fixed